riotfile.py

```diff
@@ -447,7 +447,7 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
                         "pytest-asyncio": latest,
                         "requests": latest,
                         "aiofiles": latest,
-                        "sqlalchemy": latest,
+                        "sqlalchemy": "~=1.3.0",
```

Comment:
```suggestion
                        # DEV: We can change back to `latest` after https://github.com/encode/databases/issues/298 is resolved
                        "sqlalchemy": "~=1.3.0",
```


eh? sorry I didn't think/catch this before

Reply:
Yeah I'm always torn whether to include these comments in code or leave it to be found in source control. I don't feel too strongly one way or the other.

Reply:
just thinking context is 8 months when we come back to this and wonder why we had to pin for a specific version.

like the pytest 3 issues for older versions of Flask

Reply:
yeah it should be easy enough to blame/find in git but at the same time having the comment makes it more ugly (which is good and incentivizes us to take action in the future)

Reply:
updated 🙂